src/main/java/Ingredient.java

```diff
@@ -0,0 +1,4 @@
+public class Ingredient {
+    String ingrName;
+    int ingrAmount;
```

Comment:
Jeżeli klasa nazywa się Ingredient, to nie ma potrzeby dodawać do nazw pól wewnątrz tej klasy prefixu "ingr", czy "ingredient". Wiadomo, że te elementy należą do składnika, więc wystarczy samo name i amount